pages/schools/[id].js

```diff
@@ -1,76 +1,34 @@
 import React from 'react'
-import Button from '@mui/material/Button'
 import Grid from '@mui/material/Grid'
-import Stack from '@mui/material/Stack'
 import Typography from '@mui/material/Typography'
-import { startCase } from 'lodash'
-import { useRouter } from 'next/router'
-import { isArray } from 'lodash'
 import Head from 'next/head'
 import { truncate } from 'lodash'
 import { schools, courses as allCourses } from '/lib/models'
 import { schoolShape } from '/lib/prop-types'
-import Box from '@mui/material/Box'
-import CourseCard from '/components/card'
-import { useMediaQuery } from '@mui/material'
+import CourseCard from '/components/courseCard'
 import { schoolFindById } from '/lib/models'
 
 export default function School({ school, courses }) {
-  const largeScreen = useMediaQuery('(min-width:600px)')
-  const router = useRouter()
-
-  const renderSection = (title, attr) => {
-    const displayedVal = isArray(attr) ? attr.join(', ') : attr
-
-    if (attr) {
-      return (
-        <React.Fragment>
-          <Typography variant="h6" sx={{ fontWeight: 'bold' }}>
-            {startCase(title)}
-          </Typography>
-          <Typography variant="subtitle1">{displayedVal}</Typography>
-        </React.Fragment>
-      )
-    }
-  }
-
   return (
-    <React.Fragment>
+    <>
       <Head>
-        <title>{`${school.name} - Tulsa Public Schools`}</title>
+        <title>{`${school.name} Courses - Tulsa Public Schools`}</title>
         <meta
           name="description"
           content={truncate(school.name, { length: 155 })}
         />
-        <link rel="icon" href="/images/tps-logo-color.svg" />
       </Head>
-      <Stack spacing={1}>
-        <Grid item xs={2}>
-          <Button variant="contained" onClick={() => router.back()}>
-            Go Back
-          </Button>
-        </Grid>
-        {renderSection('name', school.name)}
-        {renderSection('school_number', school.school_number)}
-      </Stack>
 
-      <Box sx={{ marginRight: '0px' }}>
+      <Typography variant="h1">{school.name}</Typography>
```

Comment:
Perhaps this should be an `h4` to take up less of the viewport and match other page headers.